backend/src/main/java/com/morak/back/auth/application/JwtTokenProvider.java

```diff
@@ -35,6 +37,7 @@ public String createToken(String payload) {
                 .compact();
     }
 
+    @Override
     public String getPayload(String token) {
```

Comment:
```suggestion
    public String parsePayload(String token) {
```

는 어떨까요 ? get이라는 이름은 그냥 있는걸 그대로 꺼내오는 듯한 느낌이에요 ☺️

Reply:
흠.. 그렇네요parse로 바꿧어요